website/sidebars.json

```diff
@@ -0,0 +1,10 @@
+{
+  "docs": {
+    "Docusaurus": ["doc1"],
```

Comment:
We will update sidebars.json. (after merge this PR?)

[Adding Documents to a Sidebar](https://docusaurus.io/docs/en/navigation.html#adding-documents-to-a-sidebar "Adding Documents to a Sidebar")
Current limitation: [Support sub-directories in docs/ · Issue #323 · facebook/Docusaurus](https://github.com/facebook/Docusaurus/issues/323 "Support sub-directories in docs/ · Issue #323 · facebook/Docusaurus")

Reply:
Ah, I completely missed side bars. We can improve details as you said. Thank you for reviewing 😄